src/lib/isDivisibleBy.js

```diff
@@ -1,7 +1,11 @@
 import assertString from './util/assertString';
 import toFloat from './toFloat';
+import isInt from './isInt';
 
 export default function isDivisibleBy(str, num) {
   assertString(str);
+  if (num.constructor !== Number && !isInt(num)) {
```

Comment:
Isn't easier to use `toInt` here instead of doing validation on the number?

Reply:
@tux-tn - i will make those changes before monday. i think that approach makes sense. thanks for the input.

Reply:
i've updated `isInt()` and `toInt()` and will update docs later if you agree with the changes.

> Isn't easier to use toInt here instead of doing validation on the number?
i thought that most people would just pass a number, not a string, as the second argument to `isDivisibleBy`. it's not clear that the divisor must be a string. (the docs imply that it's a number - `isDivisibleBy(str, number)` - so if they pass a number i thought it made sense to use that. and in theory the number doesn't need to be an integer, e.g., `isDivisibleBy('7.5', 2.5)`.

if they use floats they'll get potential rounding errors, but the way i read the docs maybe i should change the check to:
`if (num.constructor !== Number && !isFloat(num))`.

@tux-tn thoughts?

Reply:
You are right, the number doesn't have to be a natural number and checking for float/parsing to float should be good

Reply:
A last thing, can you add a test case for the new condition in order to bring back coverage to 100%?

Reply:
@tux-tn - done